pkg/collector/corechecks/cluster/ksm/kubernetes_state.go

```diff
@@ -141,6 +133,9 @@ type KSMConfig struct {
 	// LeaderSkip forces ignoring the leader election when running the check
 	// Can be useful when running the check as cluster check
 	LeaderSkip bool `yaml:"skip_leader_election"`
+
+	// Private field containing the label joins configuration built from `LabelJoins`, `LabelsAsTags` and `AnnotationsAsTags`.
```

Comment:
maybe add as comment what is the map key. from what I understand it the the metrics family name.